packages/paypal-js/src/utils.ts

```diff
@@ -66,7 +66,15 @@ export function processOptions(options: PayPalScriptOptions): {
     url: string;
     attributes: StringMap;
 } {
-    let sdkBaseUrl = "https://www.paypal.com/sdk/js";
+    const { env } = options;
```

Comment:
I liked Greg's suggestion to use `environment` but also am curious about validating this value like:

```js
if (environment) {
  if (!["sandbox", "production"].includes(environment)) {
    throw new Error('The `environment` option must be either "sandbox" or "production"')
  }
}
```

Reply:
Ah! great catch. I forgot the typescript interface isn't available in transpiled code.